apps/prairielearn/src/sync/fromDisk/courseInfo.ts

```diff
@@ -5,6 +5,14 @@ import * as infofile from '../infofile.js';
 
 const sql = sqldb.loadSqlEquiv(import.meta.url);
 
+function isExampleCourse(courseInfo: CourseData['course']['data']): boolean {
+  return (
+    courseInfo?.uuid === 'fcc5282c-a752-4146-9bd6-ee19aac53fc5' &&
+    courseInfo?.title === 'Example Course' &&
+    courseInfo?.name === 'XC 101'
```

Comment:
When this function is called, we'll already have validated that the data is present. We should adjust the type of the `courseInfo` argument here to reflect that it won't be null/undefined. Probably that just means using the `CourseJson` type.

Reply:
Addressed in 57735c1.